O/OpenCV/bundled/patches/opencv-gen.patch

```diff
@@ -0,0 +1,270 @@
+diff --git a/modules/julia/CMakeLists.txt b/modules/julia/CMakeLists.txt
```

Comment:
I am confused: as far as I can tell, `opencv-gen.patch` is not used at all, yet it is updated here?!